dotnet/Workloads/vs-workload.template.props

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project>
   <PropertyGroup>
-    <TargetName>Microsoft.NET.Sdk.MaciOS.Workload.@VSMAN_VERSION@</TargetName>
+    <TargetName>Microsoft.NET.Sdk.MacCatalyst_macOS.net70RC2.Workload.@VSMAN_VERSION@</TargetName>
```

Comment:
Could we keep this naming consistent with what was done here? https://github.com/xamarin/xamarin-macios/pull/16060/files (e.g. `Microsoft.NET.Sdk.MacCatalyst.macOS.Workload`)

Reply:
@pjcollins fixed

Reply:
Sorry I should have mentioned that we should keep the version suffix, that change was added to differentiate these manifests from the net6.0 ones that are included side by side.

Reply:
@pjcollins fixed!